backend/src/core/models/user/api-key.ts

```diff
@@ -29,4 +29,8 @@ export class ApiKey extends Model<ApiKey> {
 
   @Column({ type: DataType.DATE, allowNull: false })
   validUntil: Date
+
+  // we're enforcing not-null only for new keys via application logic
+  @Column({ type: DataType.ARRAY(DataType.STRING), allowNull: true })
+  notificationAddresses: string[]
```

Comment:
**[question]** in the future, if we expand to non-email-based methods of sending notifications (e.g. SMS), we can just add a new column right

Reply:
**[nit]** suggest renaming this to `notificationEmailAddresses`

Reply:
Oh no, I'm thinking we will be using the same column, that's why I named it `notificationAddresses` only (who knows we might include physical addresses too. JKJK lmao)

Reply:
in that case maybe `notificationContacts`? anyway small issue — just the db column harder to rename in the future, but no biggie